example/src/App.js

```diff
@@ -1,14 +1,29 @@
 import * as React from 'react';
-import {View} from 'react-native';
-import {multiply} from 'react-native-key-command';
+import {
+    registerKeyCommand, unregisterKeyCommand, constants, eventEmitter,
+} from 'react-native-key-command';
```

Comment:
We should follow our style guide
```suggestion
import * as KeyCommand from 'react-native-key-command';
```

We also have to update the references below, e.g. `KeyCommand.unregisterKeyCommand`

Reply:
done